javascript-functions-practice/get-character-at-index.js

```diff
@@ -0,0 +1,5 @@
+/* exported getCharacterAtIndex */
+
+function getCharacterAtIndex(string, index) {
+  return (string.charAt(index));
```

Comment:
This works great, and you can also do 
return string[index]

Reply:
Ah, yeah, that would be a little more efficient. Thank you!  :)